svg/elements/textpath.json

```diff
@@ -51,10 +51,12 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": true,
+                "notes": "Chrome only accept path element as a valid reference, basic shapes (rect, circle, ellipse, line, polygon, polyline) won't work."
```

Comment:
Suggested rewrite: Chrome only accepts references to path elements. Basic shapes (rect, circle, ellipse, line, polygon, polyline) won't work.

Regardless, 'accepts' needs to be plural and 'basic' is the start of a new sentence.